crates/net/network/src/transactions/fetcher.rs

```diff
@@ -800,8 +812,8 @@ impl Stream for TransactionFetcher {
             debug_assert!(
                 self.active_peers.get(&peer_id).is_some(),
                 "`%peer_id` has been removed from `@active_peers` before inflight request(s) resolved, broken invariant `@active_peers` and `@inflight_requests`,
-`%peer_id`: {},
-`@self`: {:?}",
+    `%peer_id`: {},
+    `@self`: {:?}",
```

Comment:
```suggestion
`%peer_id`: {},
`@self`: {:?}",
```
this way it's printed to stdout without indentation

Reply:
my bad! missed this!